Add theme for Rapid React (2022)

caburum opened the pull request:
- Moved year independent styles from theme-2020.css to main.css
- Added theme-2022.css
- Updated some view files to use new images/colors
<img src="https://i.imgur.com/TrOSgsW.png" />

JL102 commented:
I noticed that you both modified `theme-2020.css` and created a new file, `main.css`. Additionally, that means the `theme-2022.css` file that was created is entirely unused. Instead, could you please keep `theme-2020.css` as it previously was, and place your modified theme in `theme-2022.css`? That way, we can swap between yearly themes easily if necessary.

We _could_ still have a `main.css` for the layout-related classes that are not year-specific, like `theme-inline-padding` and `notification-card-container`. I'm definitely not against doing that if you agree.

caburum commented:
Okay, I reverted the changes to `theme-2020.css` so it can be used like it was previously. I had added 2022-specific stuff to `theme-2022.css` and moved the rest of the general styles to `main.css`.

I think that for 2022+, `main.css` should hold the things that aren't dependent on the year like general styles, and `theme-xxxx.css` should hold styles specific to that year like the game colors and overrides of the normal styling. There can be standard vars like primary that are set by each year's theme and then referenced by the standard styles so everything doesn't need to be copied every year.

JL102 commented:
Verified that the theme can be reverted by switching the stylesheet link from theme-2022.css to theme-2020.css (and also changing the associated image links).
I believe that we should either change the text color of `theme-submit` and `theme-input` back to white, or give them a lighter background color, but that can be up for discussion.
